ansible/roles/kubernetes/tasks/secrets.yml

```diff
@@ -81,15 +81,41 @@
 
 - include: gen_tokens.yml
   when: inventory_hostname == groups['masters'][0]
```

Comment:
As you mention previously, does these register variables `gentoken_master` and `gentoken_node` wouldn't got skipped values ?